hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1453,11 +1570,13 @@ on alpha3beta4 (Pym et al., 2005). However, in contrast to its
 reported transient enhancement when expressed in oocytes,
 an inhibition of alpha4beta2 has been reported when
 expressed in human SH-EP1 cells (Wu et al., 2004)."
-
+SET Confidence = "Low"
 a(CHEBI:"amyloid-beta") -| act(p(HGNC:CHRNA7))
 a(CHEBI:"amyloid-beta") -> act(complex(p(HGNC:CHRNA4), p(HGNC:CHRNB2)))
+#the line above and below seem to contrast each other. In evidence it is differentiated temporally. Is there any way to indicate that diffrence?
```

Comment:
usually you could add annotations, but as there are no exact time points mentioned in this evidence i would stick with the final result, so the blocking